README.md

```diff
@@ -1,24 +1,42 @@
 # Northcoders News API
 
-The purpose of this API is to access application data programmatically. This is mimicing the building of a real world backend service (similar to reddit).
+The purpose of this API is to access application data programmatically. This is mimicing the building of a real world backend service (similar to reddit). NC News contains articles containing a body, author, topic, votes, and comment count, along with the date it was created.
 
-To set up the database correctly you'll need the following commands:
+Available Endpoints:
```

Comment:
might be worth here just directing users to the fact that the up to date list of endpoints is available at /api - otherwise this list will have to be continuously updated!